app/src/main/res/values/strings.xml

```diff
@@ -413,6 +413,8 @@
     <string name="total_loan_savings_description">Above Loan and Savings accout balance are
         calculated on behalf of your all savings and loan accounts.
     </string>
+    <string name="cancel_transfer">Are you sure you want to cancel the transfer?</string>
```

Comment:
@luckyman000786 please add the string in all strings.xml, whenever you add a new string. Here is a medium blog(https://medium.com/@therajanmaurya/add-localization-string-in-application-supported-all-languages-using-keyboard-shortcut-807e0a388672). I have written, have a look